modules/learn/pages/services-and-indexes/services/analytics-service.adoc

```diff
@@ -112,9 +112,13 @@ This continuous data ingestion allows operational and analytic queries to run co
 
 Due to the large scale and duration of operations it is likely to perform, the Analytics Service should be run _alone_, on its node or nodes in the cluster, _with no other Couchbase Service running on the Analytics nodes_.
 
-For the practical steps required to initialize or join a cluster, and to deploy services, see
-xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
-For information on how to run analytic queries, see the xref:analytics:introduction.adoc[Introduction] to Couchbase Analytics.
+For more information about:
+
+* Initializing a cluster, see xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
```

Comment:
This should be for more info on initializing or joining a cluster (as in the original description).  Even though the page is called "Create a Cluster", it does have info on joining a cluster as well ... and once you get to that page, you can also see the list of other specific topics on joining a cluster in the navigation menu if that's what you are looking for.

`* Initializing or joining a cluster, see xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].`

Note:  For the Analytics service, it's likely to be the only service running on a node.  So, in most test and prod cases, people are likely to configure a new node with the Analytics service and join an existing cluster.   In dev, however, they may add or remove the analytics service from an existing node.